pages/home-new.jsx

```diff
@@ -231,28 +234,41 @@ export default function Home({ posts, keyWritings, keyInitiatives }) {
           ))}
         </ul>
       </div>
+       <div className="mt-10 flex">
+                <a href="https://lifeitself.org/initiatives" className="text-sm font-semibold leading-6 text-indigo-400">
+                  See More Initiatives<span aria-hidden="true">&rarr;</span>
+                </a>
     </div>
-  </section>
```

Comment:
@nathenf missing a closing section tag. Can merge and I'll push the fix to main.